misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -828,6 +831,22 @@ abstract class PathAssertionsSpec(
         }
     }
 
+    describeFun(isRelative) {
+        val isRelativeFun = isRelative.lambda
+
+        it("throws an AssertionError for absolute path") {
+            val path = tempFolder.newFile("test")
+            expect {
+                expect(path).isRelativeFun()
+            }.toThrow<AssertionError>()
```

Comment:
I think we should check the error message here to make sure the correct failure happened. Something like 

```kotlin
toThrow<AssertionError> {
  messageContains(IS.getDefault())
  messageContains(RELATIVE.getDefault())
}
```

Reply:
I would even use
```
toThrow<AssertionError> {
  messageContains("$isDescr: ${RELATIVE.getDefault()}")
}
```